src/styles/global.css

```diff
@@ -0,0 +1,15 @@
+@import url('https://fonts.googleapis.com/css?family=Lilita+One|Quicksand:300&display=swap');
+html {
+    background-color: #ffffff;
+    margin: 3% 10% 2% 10%;
```

Comment:
`#ffffff` and be shorten as `#fff` or `white`.
and margin can be shorten as `margin: 3% 10% 2%;`

And I would not recommend using % for you margin, as it is less predictable in my own opinion.